projects/plugins/jetpack/modules/contact-form/css/jquery-ui.css

```diff
@@ -0,0 +1,1314 @@
+/*! jQuery UI - v1.13.2 - 2022-07-14
```

Comment:
Do we need the full library? Could we do with only the elements we need, pulled in the existing stylesheet?

Reply:
We don't need the full library.
But I didn't find a way to 'officially' get a file with only the jQuery selectmenu styles.
Are you suggesting to hand pick the necessary styles from the main file?

Reply:
I went to the jQuery UI repo, and I found an 'official' jQuery UI stylesheet for the Selectmenu: https://github.com/jquery/jquery-ui/blob/main/themes/base/selectmenu.css
The widget has more style dependencies, but since we're overriding most styles, the file above was enough to keep it working correctly.